### PR TITLE
karthikeyan97 implementation

### PR DESCRIPTION
**Pull request by karthikeyan97**
#### Check List:
- [x] Tests pass (`./test.sh <username>` shows no differences between expected and actual outputs)
- [x] All formatting changes by the build are committed
- [x] Your launch script is named `calculate_average_<username>.sh` (make sure to match casing of your GH user name) and is executable
- [x] Output matches that of `calculate_average_baseline.sh`
* Execution time:4.5
* Execution time of reference implementation:208

### CODE REVIEW

File: prepare_karthikeyan97.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (gunnarmorling) [inline prepare_karthikeyan97.sh:1]: That file is not needed when empty.

**Comment by gunnarmorling** [comment]
Hey, thx for particpating! As per the README and the PR template, new entries should run under 10 sec at this point, as it's just getting to hard to operate otherwise. It's not a hard and fast rule, but this one comes in at 39.995 s, i.e. way beyond the threshold. So maybe move it to draft state for now and iterate a bit on your end until it's there? Thx!

**Review by gunnarmorling — CHANGES_REQUESTED** [review]
See above.

**Comment by karthikeyan97** [comment]
Hi made some changes and think this will do the magic.Can u please run this once again

**Pull request closed without merging**